src/sentry/static/sentry/app/views/settings/components/forms/controls/multipleCheckbox.jsx

```diff
@@ -0,0 +1,73 @@
+import {Box} from 'grid-emotion';
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+import {defined} from '../../../../../utils';
+
+const MultipleCheckboxWrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+const Label = styled.label`
+  font-weight: normal;
+  white-space: nowrap;
+  margin-right: 10px;
+  margin-bottom: 10px;
+  width: 20%;
+`;
+
+const CheckboxLabel = styled.span`
+  margin-left: 3px;
+`;
+
+export default class MultipleCheckbox extends React.Component {
```

Comment:
@MaxBittker Let's keep an eye out for new components like these where storybook stories would be helpful